kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -291,12 +291,7 @@ define([
         }
 
         // MESSAGE HANDLERS
-        function doAttach(node) {
-            container = node;
-            ui = UI.make({
-                node: container,
-                bus: bus,
-            });
```

Comment:
moved this all down into the `start` function so that there's no need to pass an argument